model/github_app_auth.go

```diff
@@ -69,6 +70,18 @@ func (githubAppAuth *GithubAppAuth) Upsert() error {
 	return err
 }
 
+// GetGitHubAppAuth fulfills the GithubAppAuthProvider interface and returns the app auth
+// for the given project.
+func (githubAppAuth *GithubAppAuth) GetGitHubAppAuth() *evergreen.GithubAppAuth {
```

Comment:
I don't think this function is being used

Reply:
I'll include it in a separate PR that's suppose to complete a TODO comment